Assignments/assignment2/StackImpl.java

```diff
@@ -0,0 +1,78 @@
+package assignment2;
+
+class Node
+{
+	int data;
+	Node next;
+};
+class Stack
+{
+	private Node top;
+	public Stack()
+	{
+		this.top=null;
+	}
+	public void push(int x)
+	{
+		Node node = new Node();
+		System.out.println("Inserting"+x);
+		node.data=x;
```

Comment:
Required spaces are mandatory for all the files.

Reply:
done